h/api/auth.py

```diff
@@ -13,3 +15,21 @@ def get_user(request):
                 consumer = Consumer(key)
                 return User(userid, consumer, False)
     return None
+
+
+def translate_annotation_principals(principals):
+    """
+    Translate a list of annotation principals to a list of pyramid principals.
+    """
+    result = set([])
+    for principal in principals:
+        # Ignore suspicious principals from annotations
+        if principal.startswith('system.'):
```

Comment:
What's the problem here? We're trying to avoid letting people make annotations writable by system.Everyone so that they can come back later when not logged in and edit them?